src/main/java/hudson/plugins/view/dashboard/test/TestTrendChart.java

```diff
@@ -149,18 +149,18 @@ public int compare(LocalDate d1, LocalDate d2) {
 
          if (run != null) { // execute only if job has builds
         	 LocalDate runDay = new LocalDate(
-        			 run.getTimeInMillis() - dateShift*60000, GregorianChronology.getInstanceUTC());
+        			 run.getTimeInMillis() - dateShift*60000l, GregorianChronology.getInstanceUTC());

```

Comment:
@stephenc yes it should.  (and hence why "l" as a variable name should be banned!)